tv_apps/smart-system/js/preview_window.js

```diff
@@ -90,10 +93,45 @@
     }
   };
 
+  PreviewWindow.prototype._handle_mozbrowsercontextmenu = function(evt) {
```

Comment:
- Put this logic to AppInstallManager
- Use AppModalDialog instead of BrowserContextMenu